client/gui-qt/icons.cpp

```diff
@@ -39,21 +44,84 @@ fcIcons *fcIcons::instance()
  */
 void fcIcons::drop() { NFCN_FREE(m_instance); }
 
+namespace {
+
+class icon_engine : public QIconEngine {
```

Comment:
It would be good to add a few words about what this icon engine does differently, maybe "Icon engine that loads SVG images and changes the color of black (`#000`) paths to match the text color."